src/helpers/log.ts

```diff
@@ -59,6 +60,7 @@ export const log = {
   },
 
   error: ({ title, body = "" }: { title: string; body?: string | object }) => {
+    sentryCaptureMessage(title);
```

Comment:
@quietbits has done a great hob making sure thrown errors eventually bubble up to this logging method, so let's just capture everything here